docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -611,3 +611,24 @@ to `false`, and then save the integration.
 == Hosted {agent} is offline
 
 To scale the {fleet-server} deployment, {ecloud} starts new containers or shuts down old ones when hosted {agent}s are required or no longer needed. The old {agent}s will show in the Agents list for 24 hours then automatically disappear.
+
+[discrete]
+[[hosted-agent-8-x-upgrade-fail]]
+== APM & Fleet fails to upgrade to 8.x on {ecloud}
+
+In some scenarios, upgrading APM & Fleet to 8.x may fail if the Elastic Cloud agent policy was modified manually by the user. The {fleet} app in {kib} may show a message like:
+
+[source,sh]
+----
+Unable to create package policy. Package 'apm' already exists on this agent policy
+----
+
+You can work around this problem by resetting the {ecloud} agent policy with an API call. Note: this will remove any custom integration policies that you've added to this policy, such as Synthetics monitors.
```

Comment:
```suggestion
To work around this problem, you can reset the {ecloud} agent policy with an API call. Note that this will remove any custom integration policies that you've added to the policy, such as Synthetics monitors.
```

Reply:
I'd either change the sentence as shown or make this a proper note by putting it in a new paragraph beginning with `NOTE:` (uppercase is necessary for the build to output the formatting correctly).